packages/uniorg-parse/src/parser.ts

```diff
@@ -902,7 +902,13 @@ class Parser {
     this.parseEmptyLines();
     const _end = this.r.offset();
 
-    return u('src-block', { affiliated, language, value });
+    return u('src-block', {
+      affiliated,
+      language,
+      switches: switches?.trim(),
+      parameters: parameters !== '' ? parameters?.trim() : undefined,
```

Comment:
I am not sure if this is the API that you want, but otherwise this impacts all the other tests, as parameters is an empty string when no parameters are found.

Reply:
I believe unified requires all data to be json-serializable, which means `undefined` values are not allowed. It's better to return `null` even if it means updating old tests